fr/lecons/normaliser-donnees-textuelles-python.md

```diff
@@ -91,6 +95,10 @@ En effet, la fonction `stripTags()` du module `obo.py` retourne une chaine de ca
 
 Modifier `html-to-list1.py` pour y appliquer la méthode `lower()` à `obo.stripTags(html)`&nbsp;:
 
+<div class="alert alert-warning">
+Attention&nbsp;: à cause des modifications faites au site du Old Bailey Online depuis la publication de cette leçon, le lien <code>http://www.oldbaileyonline.org/browse.jsp?id=t17800628-33&div=t17800628-33</code> ne fonctionnera plus dans le code ci-dessous. Vous avez deux options pour contourner le problème&nbsp;: si vous suivez actuellement cette leçon en utilisant un autre site qui fonctionne, vous pouvez simplement remplacer le lien du Old Bailey Online avec votre propre lien correspondant&nbsp;: 
```

Comment:
This could be an interesting solution !

Reply:
I do think it makes sense to keep the original link in the example code block, even though it won't work. I'm not sure we can be certain that `http://www.URL.org/FILEPATH` would accurately reflect all the possible URLs our readers might want to use. 

In a way, I think the code is easier to understand when it uses a concrete example, rather than a placeholder.

Hopefully, the warning boxes posted around the lesson should be enough to let readers know that they'll need to adapt the code before running it.